en/docs/install-and-setup/install-on-vm.md

```diff
@@ -67,40 +69,35 @@ Follow the instructions below to install the WSO2 API Microgateway related binar
     | Cores  | 2     |
     | OS     | Any   |
 
-2.  Log in to the command line (Terminal on Mac).
+2.  Download the [WSO2 API Microgateway distribution](https://wso2.com/api-management/api-microgateway/) based on your OS and extract it.
 
-3.  Download the [WSO2 API Microgateway distribution](https://wso2.com/api-management/api-microgateway/) based on your OS and extract it.
+3.  Extract the archive file to a dedicated directory for the WSO2 API Microgateway **Runtime**, which will hereafter be referred to as `<MGW_HOME>`.
 
-4.  Extract the archive file to a dedicated directory for the WSO2 API Microgateway **Runtime** , which will hereafter be referred to as `           <MGW_HOME>          ` .
-
-5.  Append the full path of the `           /bin          ` folder of the extracted WSO2 API Microgateway distribution to the PATH environment variable.
+4.  Append the full path of the `/bin` directory of the extracted WSO2 API Microgateway distribution to the PATH environment variable.
     This allows you to run the WSO2 API Microgateway distribution-related commands from any directory location.
+    <details>
+    <summary>Click here for more instructions on configure path variable in your OS.</summary>  
+    **Setup runtime path in Linux/MacOS**
+    
+    ``` java tab="Format"
+    export PATH=$PATH:<extracted_distribution_location>/bin
+    ```
+        
+    **Setup runtime path in Windows**
+    
+    Use one of the following approaches to define the paths.
 
-    -   **Linux and OS X**
-
-        **Example**
+    1.  Set the Microgateway toolkit distribution path using the following command in the terminal.
 
-        ``` java
-            export PATH=$PATH:<extracted_distribution_location>/bin
+        ``` java tab="Format"
+        setx "%path%;<extracted_distribution_location>/bin"
```

Comment:
setx?

Reply:
It's a windows command https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx